Payload/bleemsync/bin/execute

```diff
@@ -8,42 +8,39 @@
 ini_file="/media/bleemsync/etc/bleemsync/CFG/bleemsync_cfg.INI"
 cfg_file="/var/volatile/bleemsync.cfg"
 
-[ -f "$cfg_file" ] && rm -f "$cfg_file" && touch "$cfg_file"
-echo "#!/bin/sh" >> "$cfg_file"
-cat "$ini_file" | grep -v ";" | grep -v "\[" >> "$cfg_file"
+rm -f "$cfg_file"
+grep -v -e "^;" -e "^\[" "$ini_file" > "$cfg_file"
 
 if [ -f "$cfg_file" ]; then
-  . "$cfg_file"
+  source "$cfg_file"
 else
   echo "wtf" #Add a proper error at some point here...
 fi
 
-source "/media/bleemsync/etc/bleemsync/CFG/boot_sequence.cfg"
+source "${bleemsync_path}/etc/bleemsync/CFG/boot_sequence.cfg"
```

Comment:
Once we establish a config source, everything should be relative to what's configured.